### PR TITLE
add description to state 19 in novelan heatpump binding

### PR DESCRIPTION
**Pull request by coolchip**
The heatpump binding throws a "found new value for reverse engineering" for state 19 at my openhab system. My heatpump shows "Warmw. Nachheizung" at this time. So I added this information to the novelan binding.

### CODE REVIEW

File: bundles/binding/org.openhab.binding.novelanheatpump/src/main/java/org/openhab/binding/novelanheatpump/internal/HeatPumpBinding.java [hunk 1]
```diff
@@ -351,6 +351,8 @@ private String getExtendeStateString(int[] heatpumpValues) {
             case 17:
                 returnValue = Messages.HeatPumpBinding_ZWE_OPERATION;
                 break;
+            case 19:
+                returnValue = Messages.HeatPumpBinding_SERVICE_WATER_ADDITIONAL_HEATING;
```
Comment (watou) [inline bundles/binding/org.openhab.binding.novelanheatpump/src/main/java/org/openhab/binding/novelanheatpump/internal/HeatPumpBinding.java]: Please add a `break;` like in earlier cases, and also please re-test.  Thanks!

Comment (coolchip) [inline bundles/binding/org.openhab.binding.novelanheatpump/src/main/java/org/openhab/binding/novelanheatpump/internal/HeatPumpBinding.java]: OMG.... bad mistake. thank you for watching.

**Comment by watou** [comment]
Thanks @coolchip!

**Comment by coolchip** [comment]
Thanks @watou

**Pull request merged**